tox.ini

```diff
@@ -23,27 +25,16 @@ deps =
     pinned: six==1.9.0
     pinned: werkzeug==0.9.6
 
-    latest: eventlet  # need something
+whitelist_externals = make
 
 commands =
-    pip install --editable .[dev]
-    flake8 nameko test
-    pylint --rcfile=pylintrc nameko -E
-    coverage run --concurrency=eventlet --source nameko -m \
-        pytest test {posargs:--timeout=30}
-    coverage report --show-missing --fail-under=100
+    codebase: pip install --editable .[dev]
```

Comment:
It got ugly making the examples run with multiple python versions. You have to make a testenv that exactly matches the expanded environments so that they don't inherit the default. Like this:

```
[testenv:py27-examples]
pip install --editable .[dev,examples]
make test_examples

[testenv:py34-examples]
pip install --editable .[dev,examples]
make test_examples

...
```

It's not terrible with only two repetitions, but would get worse with more python versions or varieties of commands. FWIW I find the new version easier to understand too.